packages/integration_test/example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
```

Comment:
can we add another test to make sure the whole thing builds with gradle 5 / agp 3?

Reply:
that is already the case in the customer tests

Reply:
which customer tests are you referring to?

Reply:
https://github.com/flutter/tests/tree/master/registry - Although, it looks like we don't build apk in any of those. I will add one that does it.

Reply:
Filed https://github.com/flutter/flutter/issues/70964. Gradle 5.* is currently tested by building the Flutter Gallery demo app.